src/components/application_manager/src/resumption/resume_ctrl_impl.cc

```diff
@@ -335,6 +347,37 @@ void ResumeCtrlImpl::OnAwake() {
   StartSavePersistentDataTimer();
 }
 
+void ResumeCtrlImpl::SaveLowVoltageTime() {
+  ResetLowVoltageTime();
```

Comment:
why do this extra reset?
 low_voltage_time_ = 0;
low_voltage_time_= new value;

Reply:
@yang1070 Right. Removed in bbd93f1cf73de598eb4bbcf1760114547a19e08e